src/text-editor.coffee

```diff
@@ -866,116 +866,177 @@ class TextEditor extends Model
       @transact groupingInterval, =>
         fn(selection, index) for selection, index in @getSelectionsOrderedByBufferPosition()
 
-  # Move lines intersection the most recent selection up by one row in screen
+  # Move lines intersection the most recent selection or multiple selections up by one row in screen
```

Comment:
Thank you for the heads up, I pushed a fix on @lpommers repo, it'll be available when he'll merge the branch.